src/engine/sys/sdl_glimp.cpp

```diff
@@ -1897,7 +1897,14 @@ static bool LoadExt( int flags, bool hasExt, const char* name, bool test = true
 	{
 		if ( test )
 		{
-			logger.WithoutSuppression().Notice( "...using GL_%s", name );
+			if ( flags & ExtFlag_REQUIRED )
+			{
+				logger.WithoutSuppression().Notice( "...using required core extension GL_%s.", name );
```

Comment:
The message is wrong, taking this branch doesn't mean it must be core

Reply:
Right, fixed.